azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/ImageType.java

```diff
@@ -6,8 +6,9 @@
 
 package com.microsoft.azure.maven.webapp.configuration;
 
-public enum DockerImageType {
```

Comment:
I don't think we need change the enum type name. The original name still makes sense in the new scenario.